cmd/http-check/main.go

```diff
@@ -2,13 +2,14 @@ package main
 
 import (
 	"fmt"
+	"log"
```

Comment:
I think you should change this import back to logrus for consistency -- all of the other checks are using the package for logging: https://github.com/sirupsen/logrus